### PR TITLE
Add coding conventions to CONTRIBUTING.md

### LINKED ISSUE #11: Add a guide for contributors
Should contain:
- how to report bugs
- how to check out and build the projects
- how to use forks and pull requests
- more?

### PR DESCRIPTION
**Pull request by wallner**
This commit adds a section about code format conventions which
we would like to adhere too. It should resolve #11.

### CODE REVIEW

**Comment by tkrille** [comment]
A slightly better commit summary would be nice. Something in the sense of "Add coding conventions to CONTRIBUTING.md". Also, please, don't include the issue number in the summary, but as an additional line in the commit description, that says: resolves #11. [Only this way GitHub gets the link to the PR and can automatically close the issue](https://help.github.com/articles/closing-issues-via-commit-messages/).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,6 +32,18 @@ branch that doesn't have its origin in a pull request. We believe that not
 only does this improve the quality of our code base tremendously, but also 
 offers us a good way to learn from our mistakes.
 
+## Coding Conventions
+As coding conventions we adopted the Oracle's official [Conding conventions for
+the Java Programming Language]
+(http://www.oracle.com/technetwork/java/javase/documentation/codeconvtoc-136057.html)
+with the two follownig exceptions:
```
Comment (tkrille) [inline CONTRIBUTING.md]: Typo: `following` not `follownig`

Comment (tkrille) [inline CONTRIBUTING.md]: Also, I would write: "... the following two exceptions:"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,6 +32,18 @@ branch that doesn't have its origin in a pull request. We believe that not
...
+
+* We use a line width of 120 characters.
+* We indent with four spaces and no tabs.
+
+you can find a formatter definition for the Eclipse IDE [here].
```
Comment (tkrille) [inline CONTRIBUTING.md]: Typo: `You` not `you`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -44,6 +56,7 @@ please consider doing the following:
 * Make sure the [integration tests]
   (https://github.com/osiam/connector4java-integration-tests) pass.
 * Please squash related commits.
+* Please make sure that your pull request adheres to our coding conventions.
```
Comment (tkrille) [inline CONTRIBUTING.md]: A link to the section about the coding conventions would be nice.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -32,6 +32,18 @@ branch that doesn't have its origin in a pull request. We believe that not
 only does this improve the quality of our code base tremendously, but also 
 offers us a good way to learn from our mistakes.
 
+## Coding Conventions
+As coding conventions we adopted the Oracle's official [Conding conventions for
```
Comment (tkrille) [inline CONTRIBUTING.md]: Typo: `Coding` not `Conding`

File: CONTRIBUTING.md (hunk 2 above)
Comment (tkrille) [inline CONTRIBUTING.md]: The full stop at the end belongs after the link in the parenthesis, otherwise no link will be created.

**Comment by tkrille** [comment]
Thanks for the update @wallner. There are some typos, but I agree with the contents. After this has been merged, are you going to create a PR in each repo for this update like the last time?

File: CONTRIBUTING.md (hunk 3 above)
Comment (wallner) [inline CONTRIBUTING.md]: I wanted to do that too, Sadly I was unable to generate local anchors and link with Markdown that don't make the plain test ugly. I suspect this to be a shortcoming of markdown. As you know I am a huge fan of [Asciidoctor](http://asciidoctor.org/) anyway. But all joking aside I would be grateful for a hin on how to do that.

**Comment by wallner** [comment]
That was my original plan.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -4,17 +4,17 @@ Thank you for considering to contribute to OSIAM.
 This document is here to offer you a few helpful notes to get you started and
 make your experience as smooth as possible. They should be seen as guidelines.
 They are _not_ strict rules and we won't blame anyone who tries to contribute.
-By following the points in this document you will make our lifes a bit easier
-but we will still be happy about your contribution if you don't.
+By following the the steps in this document you will make our lives a bit
```
Comment (tkrille) [inline CONTRIBUTING.md]: Typo: doubled "the"

**Pull request merged**